### PR TITLE
added app labels where missing

### PR DESCRIPTION
**Pull request by a-thaler**
**Description**
- added app label on all functions, deployments, pods, services
- adjusted example label to example name for all resources

**Related issue(s)**
https://github.com/kyma-project/examples/issues/123

### CODE REVIEW

File: event-subscription/lambda/README.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ kubeless get-server-config
     Run this command:
 
     ```bash
-    kubeless function deploy hello-with-data --label example=event-bus-lambda-subscription --runtime nodejs8 --handler hello-with-data.main --from-file js/hello-with-data.js -n $KYMA_EXAMPLE_NS
+    kubeless function deploy hello-with-data --label example=event-subscription-lambda --runtime nodejs8 --handler hello-with-data.main --from-file js/hello-with-data.js -n $KYMA_EXAMPLE_NS
```
Comment (abbi-gaurav) [inline event-subscription/lambda/README.md]: does it also require an `app` label?

File: event-subscription/service/example-subscriber.yaml [hunk 2]
```diff
@@ -29,7 +30,7 @@ spec:
     metadata:
       labels:
         app: event-email-service
-        example: event-bus-service-subscription
+        example: event-subscription-service
```
Comment (abbi-gaurav) [inline event-subscription/service/example-subscriber.yaml]: It is more of a subscriber than a susbcription-service.

Comment (abbi-gaurav) [reply event-subscription/service/example-subscriber.yaml]: Applies to other subscription yamls also.

File: event-subscription/lambda/README.md (hunk 1 above)
Comment (a-thaler) [reply event-subscription/lambda/README.md]: very well observed, fixed it

File: event-subscription/service/example-subscriber.yaml (hunk 2 above)
Comment (a-thaler) [reply event-subscription/service/example-subscriber.yaml]: I tried to label all resources for an example with the example label using the name of the example. Here the folder name is event-subscription-service.

**Pull request merged**